packages/superset-ui-chart-controls/src/shared-controls/index.tsx

```diff
@@ -91,6 +92,8 @@ export const D3_TIME_FORMAT_OPTIONS = [
   ['%H:%M:%S', '%H:%M:%S | 01:32:10'],
 ];
 
+export const D3_TIME_FORMAT_DOCS = t('D3 time format syntax: https://github.com/d3/d3-time-format');
```

Comment:
Not within the scope of this PR, but I wonder if it's worth the trouble to make these descriptions render JSX so this could be a link. A bonus side effect would be that we only need to translate the "D3 time format syntax" string, and not maintain the URL as part of the translations codebase. I can file and/or tackle it if it seems worth pursuing.

Reply:
That's a good idea. Feel free to file a follow up PR for this!